foundation/organisation/models.py

```diff
@@ -152,11 +152,18 @@ class Board(models.Model):
     name = models.CharField(max_length=100)
     slug = models.SlugField(max_length=100, unique=True)
     description = models.TextField()
+    order = models.IntegerField(
+        blank=True, null=True,
```

Comment:
do we need the order field highlighted in brown, too?
<img width="889" src="https://user-images.githubusercontent.com/4550385/42945711-c0f858ac-8b71-11e8-8832-14d8984a7a7b.png">

Reply:
No, I guess we won't have more than one board at the same time :) I'll remove this